src/commonMain/kotlin/dev/romainguy/kotlin/math/Scalar.kt

```diff
@@ -43,3 +43,22 @@ inline fun fract(v: Float) = v % 1
 inline fun sqr(v: Float) = v * v
 
 inline fun pow(x: Float, y: Float) = (x.toDouble().pow(y.toDouble())).toFloat()
+
+fun eulerAngles(q: Float4): Float3 {
+    val x = atan2(2.0f * (q.y * q.z + q.w * q.x), q.w * q.w - q.x * q.x - q.y * q.y + q.z * q.z)
+    val y = asin(-2.0f * (q.x * q.z - q.w * q.y))
+    val z = atan2(2.0f * (q.x * q.y + q.w * q.z), q.w * q.w + q.x * q.x - q.y * q.y - q.z * q.z)
+    return transform(Float3(x, y, z), ::degrees)
+}
+
+fun quaternion(d: Float3): Float4 {
```

Comment:
I assume the `Float3` here is supposed to be 3 Euler angles?

Reply:
That's it. `d` stands for `degrees`.
Truth is that I removed the code documentation I wrote for sceneview-android just because they were no other doc in kotlin-math code.
I can add a line block on the added README.md#rotation-types section

Reply:
There's no doc because I was lazy. I'm not against new documentation :)

Reply:
lazy and because most of it is fairly obvious.

Reply:
Operator functions are just perfect for this use case but they are not `Ctrl`+`Q`ables or `Ctrl`+`Space`ables.
So maybe it's a another good reason for being lazy on this part since it will rarely be read.

Reply:
Fixed https://github.com/romainguy/kotlin-math/pull/33/commits/e2e7fd8d8293f2fc1fd1d7c296d3dd6c9135806f